iree/hal/dylib/cts/CMakeLists.txt

```diff
@@ -4,6 +4,15 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+if (IREE_BYTECODE_MODULE_FORCE_SYSTEM_DYLIB_LINKER)
```

Comment:
nit: drop the space between `if` and `(`. Same with `set(...)`

Reply:
done!